phpstan.neon

```diff
@@ -25,5 +25,4 @@ parameters:
 
     ignoreErrors:
         # collectors generics
-        - '#Parameter (.*?) of method (.*?)Collector::processNode\(\) should be compatible with parameter#'
```

Comment:
this ignore was only necessary because the types in the class below where not precise enough